requirements/edx/github.txt

```diff
@@ -74,10 +74,10 @@ git+https://github.com/edx/lettuce.git@0.2.20.002#egg=lettuce==0.2.20.002
 -e git+https://github.com/edx/event-tracking.git@0.2.1#egg=event-tracking==0.2.1
 -e git+https://github.com/edx/django-splash.git@v0.2#egg=django-splash==0.2
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
-git+https://github.com/edx/edx-ora2.git@1.4.4#egg=ora2==1.4.4
--e git+https://github.com/edx/edx-submissions.git@2.0.0#egg=edx-submissions==2.0.0
+git+https://github.com/edx/edx-ora2.git@1.4.5#egg=ora2==1.4.5
+git+https://github.com/edx/edx-submissions.git@2.0.1#egg=edx-submissions==2.0.1
 git+https://github.com/edx/ease.git@release-2015-07-14#egg=ease==0.1.3
-git+https://github.com/edx/edx-val.git@0.0.17#egg=edxval==0.0.17
+git+https://github.com/edx/edx-val.git@6ee0281419b5255c39a9a30277eb30e112eeae16#egg=edxval==0.0
```

Comment:
Once https://github.com/edx/edx-val/pull/79 is merged, I'll tag and update this dependency.